lib/snippets/release-gem

```diff
@@ -51,7 +51,8 @@ if RubygemsAPI.published?(spec.name, spec.version)
   puts "#{spec.name} version #{spec.version} is already published."
   exit 0
 else
-  Git.tag_and_push(spec.version) do
+  is_successful = Git.tag_and_push(spec.version) do
     system(*release_command)
   end
+  is_successful ? exit(0) : exit(1)
```

Comment:
Does Shipit already clearly show the error, or should we be printing "Tag and push failed"?

Reply:
Seeing the stack fail should be enough to prompt someone to observe the logs, whereby they'll see the raw output quite clearly